core/domain/acl_decorators.py

```diff
@@ -1118,3 +1119,29 @@ def test_can_change_topic_publication_status(self, **kwargs):
     test_can_change_topic_publication_status.__wrapped__ = True
 
     return test_can_change_topic_publication_status
+
+
+def get_decorator_for_accepting_suggestion(decorator):
+    """Function that takes a decorator as an argument and then applies some
```

Comment:
Add args, returns, etc. to docstring

/cc @apb7 @kevinlee12 would it be possible to check that docstrings actually have these components where necessary?

Reply:
We have a switch in place for this which will make it mandatory for all docstrings to have both these components (provided returns is not `None`) but all docstrings need to be completed before we enable this switch. I am adding a note to issue #4374 to do this once we're done with the necessary docstrings. Thanks!

Reply:
Missed these. Will add them.

Reply:
Done.